apps/ui/lev-help.md

```diff
@@ -0,0 +1,72 @@
+# Help with LEV
```

Comment:
Don't 'love' the title (and when it's used with link below) - I think something like LEV Support would be better but others might have thoughts.

Reply:
Changed to 'Help using LEV' as discussed - this is also the title GovUK use for their help pages.